scripts/run-prow-test.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# This script runs e2e tests on the AWS Load Balancer Controller
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GINKGO_TEST_BUILD="$SCRIPT_DIR/../test/build"
+
+source "$SCRIPT_DIR"/lib/common.sh
+
+function toggle_windows_scheduling(){
+  schedule=$1
+  nodes=$(kubectl get nodes -l kubernetes.io/os=windows | tail -n +2 | cut -d' ' -f1)
+  for n in $nodes
+  do
+    kubectl $schedule $n
+  done
+}
+
+echo "Cordon off windows nodes"
+toggle_windows_scheduling "cordon"
+
+eksctl utils associate-iam-oidc-provider \
```

Comment:
shall we just change our prow cluster's setup to have this enabled by default?

Reply:
vpc-cni and vpc-rc do need windows. Only for load balancer we need to disable it, so based on my earlier discussion with @abhipth decided to have it here.